### PR TITLE
Add even more test cases

### PR DESCRIPTION
**Pull request by skosito**

### CODE REVIEW

File: baseledger/x/bridge/keeper/price_and_worktoken_calculator.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ func CalculateAmountOfWorkTokens(worktokenEurPrice *big.Int, depositedUbtAmount
 
 	amountOfWorkTokensCeiled := ceilAmount(amountOfWorkTokens)
 
-	return new(big.Int).Div(amountOfWorkTokensCeiled, big.NewInt(1000000000000000000))
+	return new(big.Int).Div(amountOfWorkTokensCeiled, big.NewInt(100000000))
```
Comment (skosito) [inline baseledger/x/bridge/keeper/price_and_worktoken_calculator.go:26]: @ognjenkurtic  you merge first then i will fix this

File: baseledger/x/proof/client/rest/txutil.go [hunk 2]
```diff
@@ -85,15 +85,10 @@ func getKey(keyringInstance keyring.Keyring) (keyring.Info, error) {
 func NewKeyringInstance() (keyring.Keyring, error) {
 	input := &bytes.Buffer{}
 	fmt.Fprintln(input, viper.GetString("KEYRING_PASSWORD"))
+	fmt.Fprintln(input, viper.GetString("KEYRING_PASSWORD"))
```
Comment (skosito) [inline baseledger/x/proof/client/rest/txutil.go:88]: adding it 3 times because of prompt repeat (2 should be enough, but 3 just in case)

Comment (skosito) [reply baseledger/x/proof/client/rest/txutil.go:88]: also reminder, we need to use KEYRING_DIR and KEYRING_PASSWORD env variables

Comment (skosito) [reply baseledger/x/proof/client/rest/txutil.go:88]: viper is picking them up if we just go export VAR=value.. so should be fine

File: tests/run-testnet.sh [hunk 3]
```diff
@@ -42,6 +42,15 @@ do
     VALIDATOR_PHRASE=$(sed "$y q;d" ./validator-phrases)
     ORCHESTRATOR_PHRASE=$(sed "$y q;d" ./orchestrator-phrases)
 
+    # re-add same key to file keyring so it can be used for baseledger transactions
+    echo $ORCHESTRATOR_PHRASE >> testkey.txt
+    echo "12345678" >> testkey.txt
+    echo "12345678" >> testkey.txt
```
Comment (ognjenkurtic) [inline tests/run-testnet.sh:48]: duplicate line

File: tests/test/baseledger.test.js [hunk 4]
```diff
@@ -153,11 +153,11 @@ describe('attestations observed', async function() {
 
     parsedResponse = JSON.parse(accountBalance.text);
 
-    // check that balance increased by 1
+    // check that balance is increased
     const workTokenBalanceAfter = parsedResponse.balance.amount;
     console.log('work tokens after deposit ', workTokenBalanceAfter);
 
-    expect(+workTokenBalanceAfter).to.be.equal(+workTokenBalanceBefore + 1);
+    expect(+workTokenBalanceBefore).to.be.below(+workTokenBalanceAfter);
```
Comment (ognjenkurtic) [inline tests/test/baseledger.test.js:160]: do we want to assert exact increase amount here?

**Review by ognjenkurtic — APPROVED** [review]

File: tests/run-testnet.sh (hunk 3 above)
Comment (skosito) [reply tests/run-testnet.sh:48]: not duplicate, it asks for confirmation

File: tests/test/baseledger.test.js (hunk 4 above)
Comment (skosito) [reply tests/test/baseledger.test.js:160]: not possible due to price change

**Pull request merged**